webui/tests/tests/StepImplementation.ts

```diff
@@ -45,9 +46,14 @@ const HOST = process.env.DET_MASTER || 'localhost:8080';
 const BASE_PATH = process.env.PUBLIC_URL || '/det';
 const BASE_URL = `${HOST}${BASE_PATH}`;
 const SCREENCAST = process.env.SCREENCAST === 'true';
+const viewports = {
+  mobile: { width: 411, height: 823 },
+  desktop: { width: 960, height: 1536 },
+};
 
 function handleError(e) {
   console.error(e);
+  throw e;
```

Comment:
you are 100% correct, confirmed that we do need to throw the error in order for gauge to report and handle it. `console.error` might be unnecessary then since gauge will report it?

Reply:
good point :) as discussed I'm adding that change https://github.com/determined-ai/determined/pull/1657 so I'll wait to land that before this one